backend/cellguide/pipeline/ontology_tree/tree_builder.py

```diff
@@ -247,8 +252,11 @@ def _traverse_ontology_with_counting(
             all_unique_nodes=all_unique_nodes,
         )
 
+    def get_ontology_tree(self) -> OntologyTree:
+        return self.ontology_graph
+
     ### Get the ontology tree state
-    def get_ontology_tree_state_per_celltype(self) -> Dict[str, Any]:
+    def get_ontology_tree_state_per_celltype(self) -> Dict[str, OntologyTreeStatePerCellType]:
```

Comment:
this comment is applicable to all of the functions where the return type changed, but i believe you need to create an actual instance of the return type. so in this function, you'd need to have this be
```
                all_states_per_cell_type[end_node] = OntologyTreeStatePerCellType(
                    "isExpandedNodes": isExpandedNodes,
                    "notShownWhenExpandedNodes": notShownWhenExpanded,
                )
```